recipes/libtool/all/patches/2.4.6-0001-libtool-relocatable.patch

```diff
@@ -3,20 +3,27 @@
 
 --- libtoolize.in
 +++ libtoolize.in
-@@ -1901,11 +1901,11 @@
+@@ -1901,11 +1901,18 @@
    pkgmacro_files="@pkgmacro_files@"
 
++  if [ "$(uname -s)" = "Darwin" ]; then
```

Comment:
I think this may be due to bash/zsh things, rather than MacOS specific - I have some fixes for this, will post shortly

Reply:
Hi @jcar87,
I believe the lack of support for the UNIX command realpath(1) is a well-known limitation of BSD UNIX, which MacOS is derived from. Fortunately, it appears that Apple has added it with the recent release of Ventura. I have updated the logic here to use the OS-provided version when available. Thanks

Reply:
These are the contents of the patch I'm working with at the moment:

```diff
diff --git a/Makefile.in b/Makefile.in
index 29db7be..3046cdf 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -2423,10 +2423,10 @@ $(notes_txt): $(notes_texi)
 # files created in the build tree, so instead we regenerate the
 # manual pages if the sources for the build-tree files we want to
 # run have changed.
-$(libtool_1): $(ltmain_sh)
-	$(AM_V_GEN)$(update_mans) --help-option=--help-all libtool
-$(libtoolize_1): $(libtoolize_in)
-	$(AM_V_GEN)$(update_mans) libtoolize
+#$(libtool_1): $(ltmain_sh)
+#	$(AM_V_GEN)$(update_mans) --help-option=--help-all libtool
+#$(libtoolize_1): $(libtoolize_in)
+#	$(AM_V_GEN)$(update_mans) libtoolize
 
 install-data-local: $(lt_Makefile_in) install-scripts-local
 	@$(NORMAL_INSTALL)
diff --git a/libtoolize.in b/libtoolize.in
index 0c40fed..e82e467 100644
--- a/libtoolize.in
+++ b/libtoolize.in
@@ -1903,11 +1903,12 @@ func_require_seen_libtool ()
   pkgmacro_files="@pkgmacro_files@"
 
   # Locations for important files:
-  prefix="@prefix@"
-  datadir="@datadir@"
-  pkgauxdir="@pkgauxdir@"
-  pkgltdldir="@pkgdatadir@"
-  aclocaldir="@aclocaldir@"
+  scriptdir=$(CDPATH= cd -- "$(dirname -- "$0")" && pwd)
+  prefix="$scriptdir/.."
+  datadir="$scriptdir/../res"
+  pkgauxdir="$scriptdir/../res/libtool/build-aux"
+  pkgltdldir="$scriptdir/../res/libtool"
+  aclocaldir="$scriptdir/../res/aclocal"
 
   # Allow the user to override the master libtoolize repository:
   if test -n "$_lt_pkgdatadir"; then
```

this seems to work on the platforms I've tested it without having to use a different case on macOS - although I need to test it on an older version of macOS as well.